Makefile

```diff
@@ -375,6 +377,7 @@ cluster/prepare/dms:
 .PHONY: cluster/prepare/addon-params
 cluster/prepare/addon-params:
 	@-oc process -n $(NAMESPACE) QUOTA=$(DEV_QUOTA) StsRoleARN=$(ROLE_ARN) -f config/secrets/addon-params-secret.yaml | oc apply -f -
+#	@-oc process -n $(NAMESPACE) S3_BUCKET_CREDENTIAL_USR=$(S3_BUCKET_CREDENTIAL_USR) S3_BUCKET_CREDENTIAL_PSW=$(S3_BUCKET_CREDENTIAL_PSW) -f config/secrets/addon-params-s3-bucket-secret.yaml | oc apply -f -
```

Comment:
@laurafitzgerald , thank you, I will place the credentials into addon-params-secret. 
*** I will create new branch/PR (as multiple files added in current PR because feature branch rebase ***
Will apply fix there. Thanks again